ssl/test/runner/runner.go

```diff
@@ -19571,6 +19567,52 @@ func checkTests() {
 	}
 }
 
+func cipherSuitesContains(cipherSuites []uint16, id uint16) bool {
+	for _, cipherSuite := range cipherSuites {
+		if cipherSuite == id {
+			return true
+		}
+	}
+	return false
+}
+
+func flagsContains(flags []string, subStr string) bool {
+	for _, str := range flags {
+		if str == subStr {
+			return true
+		}
+	}
+	return false
+}
+
+func fixUpCipherSuites() {
+
+	// Iterate by reference
+	for index, _ := range testCases {
+		var add_disabld_cipher_suite bool
+		var disabled_ciphers string
+
+		if cipherSuitesContains(testCases[index].config.CipherSuites, TLS_RSA_WITH_NULL_SHA) ||
+			cipherSuitesContains(testCases[index].renegotiateCiphers, TLS_RSA_WITH_NULL_SHA) {
+			add_disabld_cipher_suite = true
+			disabled_ciphers = ":NULL-SHA"
+		}
+		if cipherSuitesContains(testCases[index].config.CipherSuites, TLS_RSA_WITH_3DES_EDE_CBC_SHA) ||
+			cipherSuitesContains(testCases[index].renegotiateCiphers, TLS_RSA_WITH_3DES_EDE_CBC_SHA) {
+			add_disabld_cipher_suite = true
+			disabled_ciphers = disabled_ciphers + ":3DES"
+		}
+
+		if add_disabld_cipher_suite && flagsContains(testCases[index].flags, "-cipher") {
+			panic(fmt.Sprintf("Test %q duplicates -cipher in arguments", testCases[index].name))
```

Comment:
Style wise you should return an `error` from this function here, and then maybe panic in the main function if this returns a error from this function. Usually reserve panics for as high in the stack as possible.